ngmix/tests/test_em.py

```diff
@@ -0,0 +1,198 @@
+import pytest
+import numpy as np
+
+from ngmix.em import fit_em
+from ._sims import get_ngauss_obs
+
+FRAC_TOL = 0.001
+
+
+def randomize_gmix(rng, gmix, pixel_scale):
+    gm_data = gmix.get_data()
```

Comment:
I assume this returns a reference and not a copy?

Reply:
We have not make the gmix objects non-writeable like we did for observations. Its probably not a bad idea

Reply:
I opened an issue #133